internal/core/src/storage/MinioChunkManager.h

```diff
@@ -30,6 +42,9 @@
 
 namespace milvus::storage {
 
+enum RemoteStorageType { STORAGE_S3 = 0, STORAGE_GOOGLE_CLOUD = 1 };
```

Comment:
0 is not recommended as the valid enum value

Reply:
why?  0 is also default value of enum if you not set it